src/models/db/accounts.js

```diff
@@ -30,7 +30,11 @@ function * _upsertAccount (account, options) {
     existingAccount.setData(account)
     yield existingAccount.save(options)
   } else {
-    yield Account.create(account, options)
+    if (!account.balance) {
+      yield Account.create(_.merge({}, account, {balance: 0}), options)
+    } else {
+      yield Account.create(account, options)
+    }
```

Comment:
It is not difficult, but that logic is in five-bells-shared, so it would require releasing a new version of that component, then including it here. I guess I can do that for now, since that would allow us to put in the not null constraint. When we merge this to master, the db code will be in this repo, so it will be more straightforward.